tests/path/win32.c

```diff
@@ -145,6 +145,27 @@ void test_canonicalize(const wchar_t *in, const wchar_t *expected)
 #endif
 }
 
+void static test_canonicalize_path(const wchar_t *in, const wchar_t *expected)
```

Comment:
These here are switched ;)

```suggestion
static void test_canonicalize_path(const wchar_t *in, const wchar_t *expected)
```